sql/item_mods.sql

```diff
@@ -34821,6 +34821,7 @@ INSERT INTO `item_mods` VALUES (17740,5,15); -- MP: 15
 INSERT INTO `item_mods` VALUES (17745,3,3);  -- HPP: 3
 INSERT INTO `item_mods` VALUES (17745,13,5); -- MND: 5
 INSERT INTO `item_mods` VALUES (17745,14,5); -- CHR: 5
+-- INSERT INTO `item_mods` VALUES (17745,,); -- Additional effect: HP or MP Drain
```

Comment:
This should be the enum name of the relevant mod

Reply:
There is no mod that can handle this. Why I noted it but commented it out.

Reply:
> There is no mod that can handle this. Why I noted it but commented it out.

work in progress on this, but its going to shift every add effect type 8 and above by 1